### ISSUE TITLE
Javadoc comment formatting

### ISSUE THREAD

**Issue by acapper** [DontShaveTheYak/groovy-guru#52]
**VSCode Version:** 1.71
**Groovy-Guru Version:** 0.6.0

**Issue:**
Additional * in tooltip comment
Text on new line is not reflected in the comment

![image](https://user-images.githubusercontent.com/16818057/188924856-868238ff-5f6a-4443-aa27-7806cf81e84e.png)

**Expect:**
The comments beginning * should not be present in the tooltip comment
Text that is separated by a new line should be on a new line in the tooltip comment

**Example Code:**
```
class MyClass {
    /**
    * Test
    * Newline
    * @author test
    */
    void test(String test) {
        println('test' + test)
    }
}
```

**Replication:**
I can replicate this issue 100% of the time after version 0.5.0
This issue doesn't seem to be present in 0.4.0 although the comments dont seem quite right in that version either since everything is in a nested bullet point list

**Comment by shadycuz** [comment]
Yeah I recently updated the LS. I'll try to fix this in a week or two. I'll either fix the LS or revert it completely.

Until now you should try and downgrade to 0.5, but if I recall vscode trys to update extensions when ever it can
